src/main/java/com/pub/wysio/controller/CustomerController.java

```diff
@@ -0,0 +1,49 @@
+package com.pub.wysio.controller;
+
+import com.pub.wysio.entity.Customer;
+import com.pub.wysio.service.CustomerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class CustomerController {
+    @Autowired
+    private CustomerService service;
+
+    @PostMapping("/addCustomer")
+    public Customer addCustomer(@RequestBody Customer customer) {
+        return service.saveCustomer(customer);
+    }
+
+    @PostMapping("/addCustomers")
+    public List<Customer> addCustomers(@RequestBody List<Customer> customers) {
```

Comment:
we don't really need a batch create endpoint. As this is not part of the specification, this can be removed